.storybook/addons.js

```diff
@@ -2,4 +2,4 @@ import '@storybook/addon-actions/register'
 import '@storybook/addon-a11y/register'
 import '@storybook/addon-storysource/register'
 import '@storybook/addon-viewport/register'
-import '@storybook/addon-knobs/register'
+import '@storybook/addon-knobs/dist/register'
```

Comment:
This plug-in was actually deprecated though support is still provided. The upgrade, for some reason, moved the `register` function to the `dist` folder.

Reply:
Do you think we should drop knobs support altogether? Or should we consider that out of scope for this PR?

Reply:
Hmm. Good question. I couldn't find any explicit configuration of knobs when I looked so I'll look into removing it, since it looks like it was added "for the future".